### PR TITLE
bugfix: fix linking in fi- & sv- page

### LINKED ISSUE #65: Fix linking in fi- and sv-versions
**Describe the bug**
When using the fi and sv-versions, the link in the header image leads to `/` instead of `/fi` or `/sv`.

**To Reproduce**
1. Go to `/fi` or `/sv`
2. Click the header image
3. You are now in the english site (so, `turkupy.github.io` or `localhost:8000` depending where you test)

**Expected behavior**
The link would lead to the correct site (so, `/fi` or  `/sv`)

**Suggestion to fix**
In the `Layout`-component, use the passed `langCode` to determinate if the link wrapping the header image is `/` or `/sv`or `/fi`. 

so for example:

```js
const indexPath = langCode === 'en' ? '/' : `/${langCode}`
```

### PR DESCRIPTION
**Pull request by fandok**
Resolves #65 .

**Why this pull request exists?**
To solve the related issue

**Changes made in this pull request:**
- add validation in image for linking the page
- passed langCode in fi-
- passed langCode in sv-

@turkupy/turkypy-maintainers

### CODE REVIEW

File: src/components/Layout.jsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ const Layout = ({ children, langCode }) => {
           <SkipLink href="#main">
             {getTranslation(langCode, "header.skiplink")}
           </SkipLink>
-          <Link to="/">
+          <Link to={langCode === "en" ? "/" : `/${langCode}`}>
```
Comment (eevajonnapanula) [inline src/components/Layout.jsx:75]: I have a small request to make the code to be more readable; Could you move this into a variable, that gets passed to the `to`-attribute? So something like 

```
const linkToIndex = langCode === "en" ? "/" : `/${langCode}`
```
and then
```
<Link to={linkToIndex}>
```

**Review by eevajonnapanula — CHANGES_REQUESTED** [review]
Looks good, thank you for your work! 🎉  There is one request to be changed, to make the code more readable. Please see the comment

File: src/components/Layout.jsx (hunk 1 above)
Comment (fandok) [reply src/components/Layout.jsx:75]: okaaaay, on it

Comment (fandok) [reply src/components/Layout.jsx]: hi @eevajonnapanula i already fix it as your code review

**Comment by eevajonnapanula** [comment]
Thank you for your contribution! 😊

**Pull request merged**